packages/host/tests/acceptance/code-submode/field-playground-test.gts

```diff
@@ -998,18 +997,18 @@ module('Acceptance | code-submode | field playground', function (_hooks) {
 
   module('multiple realms', function (hooks) {
     let realm: Realm;
-    let personalRealmURL: string;
-    let additionalRealmURL: string;
+    let origin = new URL(resolvedBaseRealmURL).origin;
+    let personalRealmURL = `${origin}/testuser/personal/`;
+    let additionalRealmURL = `${origin}/testuser/aaa/`; // writeable realm that is lexically before the personal realm
```

Comment:
if you look at where the realm-server service derives it's `url` property, for the test environment it is literally just the origin of the `resolvedBaseRealmURL` config setting for the test env. no need to use the realm server service to get this, we can simply import this from the config.